scheduler/api/thrift/scoot.thrift

```diff
@@ -158,4 +158,32 @@ service CloudScoot {
     1: InvalidRequest ir
     2: ScootServerError err
   )
+  map<string, i32> GetClassLoadPcts() throws (
```

Comment:
Is Pcts a shortening of Percents? I recommend avoiding shortenings for public apis.